File: 2129-capitalize-the-title/2129-capitalize-the-title.py
class Solution:
    def capitalizeTitle(self, title: str) -> str:
        words = title.split(' ')
        for i in range(len(words)):
            words[i] = words[i].lower() if len(words[i]) <= 2 else words[i].capitalize()

        return ' '.join(words)
        

File: 2129-capitalize-the-title/2129-capitalize-the-title.rb
# @param {String} title
# @return {String}
def capitalize_title(title)
    words = title.split(' ')
    words.each_index do |i|
        words[i].size <=2 ? words[i].downcase! : (words[i].downcase!; words[i][0] = words[i][0].upcase)
    end
    words.join(' ')
end